Code/GraphMol/AddHs.cpp

```diff
@@ -1263,5 +1269,50 @@ bool needsHs(const ROMol &mol) {
   return false;
 }
 
+bool hasQueryHs(const ROMol &mol, bool unmergableOnly) {
+  int unmergableHs = 0;
+  int mergableHs = 0;
+  // We don't care about announcing ORs or other items during isQueryH
+  RDLog::LogStateSetter blocker;
+
+  for (const auto &atom : mol.atoms()) {
+    switch (isQueryH(atom)) {
+      case HydrogenType::QueryHydrogen:
+        mergableHs += 1;
+        break;
+      case HydrogenType::UnMergableQueryHydrogen:
+        unmergableHs += 1;
+        break;
+      case HydrogenType::NotAHydrogen:
```

Comment:
pls add a default too for the case that the enum class changes at some point in the future
```suggestion
      case HydrogenType::NotAHydrogen:
      default:
```